test/parallel/test_tensorflow.py

```diff
@@ -4131,8 +4130,11 @@ def test_horovod_syncbn_cpu(self):
             for x in x_list:
                 bn = tf.keras.layers.BatchNormalization(axis=1, fused=False)
                 sync_bn = hvd.SyncBatchNormalization(axis=1)
-                bn_func = bn.apply(x, training=True)
-                sync_bn_func = sync_bn.apply(tf.expand_dims(x[hvd.rank()], 0), training=True)
+                # apply() API is removed in TF 2.9.0
+                bn_func = bn.apply(x, training=True) if LooseVersion(
+                    tf.__version__) < LooseVersion('2.9.0') else bn(x, training=True)
```

Comment:
I don't think you need the version check since __call__ is by default always there in keras.layers.